Sprint-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,91 @@
 const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+  "./assets/lifeofpi.jpg",
+  "./assets/schindlerslist.png",
+  "./assets/thematrixreloaded.jpg",
+  "./assets/thefountain.jpg",
+  "./assets/thetrumanshow.jpg",
+  "./assets/inception.jpg",
+  "./assets/vforvendetta.jpg",
+  "./assets/pulpfiction.jpg",
+  "./assets/theshining.jpg",
+  "./assets/et.jpg",
+  "./assets/breakfastattiffanys.jpg",
+  "./assets/interstellar.jpg",
+  "./assets/eternalsunshineofthespotlessmind.jpg",
+  "./assets/thegrandbudapesthotel.jpg",
 ];
 
+let currentIndex = 0;
+let autoPlayInterval = null;
+let intervalTime = 500;
 
-// Write your code here
\ No newline at end of file
+const imgElement = document.getElementById("carousel-img");
+const forwardBtn = document.getElementById("forward-btn");
+const backwardBtn = document.getElementById("backward-btn");
+const autoForwardBtn = document.getElementById("auto-forward");
+const autoBackBtn = document.getElementById("auto-backward");
+const stopBtn = document.getElementById("stop");
+const delayInput = document.getElementById("delay-input");
+const applyDelayBtn = document.getElementById("apply-delay");
+
+const updateImage = () => {
+  imgElement.src = images[currentIndex];
+};
+
+forwardBtn.addEventListener("click", () => {
+  currentIndex = (currentIndex + 1) % images.length;
+  updateImage();
+});
+
+backwardBtn.addEventListener("click", () => {
+  currentIndex = (currentIndex - 1 + images.length) % images.length;
+  updateImage();
+});
+
+autoForwardBtn.addEventListener("click", () => {
+  stopAutoPlay();
+  disableAutoButtons();
```

Comment:
It seems one of the statements in line 46 and 47 is "not very useful".

You can try comment out one of them and see which resulting effect you prefer.

If you choose to disable buttons, it might be a good idea to make them look different from active buttons.